src/app/view/components/_common/iframe/iframe-porlet-legacy/iframe-porlet-legacy.component.ts

```diff
@@ -35,7 +35,9 @@ export class IframePortletLegacyComponent implements OnInit {
 
     ngOnInit(): void {
         this.dotRouterService.portletReload$.subscribe((portletId: string) => {
-            this.reloadIframePortlet(portletId);
+            if (this.dotRouterService.isJSPPortlet()) {
```

Comment:
@hmoreras  do we really need to reload the iframe on every JSP porlet or only on the `Content Type` porlet/page